web_src/js/features/install.js

```diff
@@ -40,6 +48,18 @@ export function initInstall() {
     } // else: for SQLite3, the default path is always prepared by backend code (setting)
   }).trigger('change');
 
+  const $appUrl = $('#app_url');
+  const configAppUrl = $appUrl.val();
+  if (configAppUrl.includes('://localhost') || configAppUrl.includes('://127.0.0.1')) {
```

Comment:
Missing IPv6 localhost `::1`

Reply:
IMO seldom people would do so, and the default value from config is just `localhost` (actually we even do not need to care about `127.0.0.1`, I just put it there for no strong reason).

Reply:
I can't accept such an answer. It's just ignorant to ignore IPv6 in 2023.

Reply:
Then let's only keep the localhost, because `Domain = sec.Key("DOMAIN").MustString("localhost")`

Reply:
Could just have a little helper function. It would be useful in other places as well I suppose:

````js
function isLocalhost(hostname) {
  if (hostname === 'localhost') return true;
  if (hostname === '::1') return true;
  if (/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/.exec(hostname)) return true;
  return false;
}
````

Reply:
Really no necessary to do so .... because the default domain is localhost, the default app_url is http://localhost:3000